packages/react-app/src/views/ChallengeReviewView.jsx

```diff
@@ -0,0 +1,51 @@
+import React, { useEffect } from "react";
+import axios from "axios";
+import ChallengeReviewList from "../components/ChallengeReviewList";
+
+export default function ChallengeReviewView({ serverUrl }) {
+  const [challenges, setChallenges] = React.useState([]);
+  const [isLoading, setIsLoading] = React.useState(true);
+
+  async function fetchSubmittedChallenges() {
+    setIsLoading(true);
+    console.log("getting challenges");
+    const fetchedChallenges = await axios.get(serverUrl + `challenges`, { params: { status: "SUBMITTED" } });
+    setChallenges(fetchedChallenges.data);
+    console.log(fetchedChallenges.data);
+    setIsLoading(false);
+  }
+
+  useEffect(() => {
+    fetchSubmittedChallenges();
+  }, [serverUrl]);
+
+  async function handleApprove(userAddress, challengeId, comment) {
+    console.log(`approve ${challengeId} for ${userAddress} with comment ${comment}`);
+    await axios.patch(serverUrl + `challenges`, {
+      params: { userAddress, challengeId, comment, newStatus: "ACCEPTED" },
+    });
+    await fetchSubmittedChallenges();
+  }
+
+  async function handleReject(userAddress, challengeId, comment) {
+    console.log(`reject ${challengeId} for ${userAddress} with comment ${comment}`);
+    await axios.patch(serverUrl + `challenges`, {
+      params: { userAddress, challengeId, comment, newStatus: "REJECTED" },
+    });
+    fetchSubmittedChallenges();
+  }
+
+  return (
+    <div className="container">
+      <h1>Challenge Submissions Ready for Review!</h1>
+      <div style={{ textAlign: "start" }}>
+        <ChallengeReviewList
+          challengeSubmissions={challenges}
+          isLoading={isLoading}
+          approveClick={handleApprove}
+          rejectClick={handleReject}
```

Comment:
These two don't work for me if you don't write any comment. I think it should be non-mandatory.

It seems that the request is being made, but it remain `pending`.

![image](https://user-images.githubusercontent.com/2486142/133972732-613b11b7-1cac-465e-9eea-4478a4de182a.png)